src/Main.java

```diff
@@ -1,6 +1,132 @@
 public class Main {
-    public static void main(String[] args) {
 
-        System.out.println("Hello world!");
+    public static Employee[] Employee = new Employee[10];
```

Comment:
```suggestion
    private static Employee[] employees = new Employee[10];
```
названия полей надо писать с маленькой буквы